### DISCUSSION TITLE
https://mokkapps.de/blog/simpler-two-way-binding-in-vue-with-define-model

### DISCUSSION THREAD

**Discussion by giscus** [Mokkapps/mokkapps-de-discussions#7] [Announcements]
# https://mokkapps.de/blog/simpler-two-way-binding-in-vue-with-define-model

The new defineModel compiler macro has available since Vue 3.4 and makes two-way binding much simpler and cleaner.

https://mokkapps.de/blog/simpler-two-way-binding-in-vue-with-define-model

**Comment by Kcko** [comment]
Nice explanation like always.
